tests/CMakeLists.txt

```diff
@@ -44,6 +44,7 @@ target_link_libraries(test_libs INTERFACE
 
 if (ENABLE_ONIONREQ)
     target_link_libraries(test_libs INTERFACE libsession::onionreq)
+    target_include_directories(test_libs INTERFACE ${CMAKE_BINARY_DIR}/static-deps/include)
```

Comment:
A better fix for this would be to change PRIVATE to PUBLIC inside src/CMakeLists.txt in this line:
```cmake
        target_include_directories(onionreq PRIVATE ${CMAKE_BINARY_DIR}/static-deps/include)
```
so that the onionreq target properly carries the include dependency to targets using it.

Reply:
Ah nice - I've pushed up that change 🙏